ui/src/app/applications/components/application-status-panel/application-status-panel.tsx

```diff
@@ -32,16 +33,11 @@ const sectionLabel = (info: SectionInfo) => (
     </label>
 );
 
-const sectionHeader = (info: SectionInfo, hasMultipleSources: boolean, onClick?: () => any) => {
+const sectionHeader = (info: SectionInfo, onClick?: () => any) => {
     return (
         <div style={{display: 'flex', alignItems: 'center', marginBottom: '0.5em'}}>
             {sectionLabel(info)}
-            {onClick && (
-                <button className='application-status-panel__more-button' onClick={onClick} disabled={hasMultipleSources}>
-                    {hasMultipleSources && helpTip('More details are not supported for apps with multiple sources')}
-                    <i className='fa fa-ellipsis-h' />
-                </button>
-            )}
+            {onClick && <button className='application-status-panel__more-button' />}
```

Comment:
There was a 'More' button here before Ishita disabled it. Is this the new intended behavior?